Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackParamsEstimationAlgorithm.hpp

```diff
@@ -46,8 +45,11 @@ class TrackParamsEstimationAlgorithm final : public IAlgorithm {
   struct Config {
     /// Input seeds collection.
     std::string inputSeeds;
-    /// Input prototracks (optional)
+    /// Input prototracks (optional).
     std::string inputProtoTracks;
+    /// Input particle hypothesis (optional). If not given, the static particle
+    /// hypothesis from the config is used.
+    std::string inputParticleHypotheses;
```

Comment:
We could consider making this an `std::variant` here in the config to make it clear that they're mutually exclusive. I think pybind11 can handle this.

Alternatively, I started explicitly using `std::optional<std::string>` to model optional properties instead of using the empty string as a sentinal.